config/default.json

```diff
@@ -358,7 +358,8 @@
     "ExperimentalSettings": {
         "ClientSideCertEnable": false,
         "ClientSideCertCheck": "secondary",
-        "EnablePostMetadata": false
+        "EnablePostMetadata": false,
+        "LinkMetadataTimeout": 5000
```

Comment:
Is it worth tracking changes to this with analytics?

Reply:
Good call. I don't know if we'll ever look at it, but it's probably good to have just in case